test/manifest.cpp

```diff
@@ -82,61 +82,57 @@ static const auto manifest_json = std::string(R"(
   ]
 })");
 
+// Quicr namespaces encoded according to the URL template in the manifest
+static const auto ns_vid_a_1 = quicr::Namespace(0x0000010100000dc00001000000000000_name, 80);
+static const auto ns_vid_a_2 = quicr::Namespace(0x0000010100000dc10001000000000000_name, 80);
+static const auto ns_vid_a_3 = quicr::Namespace(0x0000010100000dc20001000000000000_name, 80);
+static const auto ns_aud_a_1 = quicr::Namespace(0x0000010100000d010001000000000000_name, 80);
```

Comment:
These can be constexpr

Reply:
Prefer unamed namespace to multiple statics:
```suggestion
namespace {
constexpr auto ns_vid_a_1 = quicr::Namespace(0x0000010100000dc00001000000000000_name, 80);
constexpr auto ns_vid_a_2 = quicr::Namespace(0x0000010100000dc10001000000000000_name, 80);
constexpr auto ns_vid_a_3 = quicr::Namespace(0x0000010100000dc20001000000000000_name, 80);
constexpr auto ns_aud_a_1 = quicr::Namespace(0x0000010100000d010001000000000000_name, 80);
}
```
(Note: Sample code has not been run through clang-format)

Reply:
Addressed in https://github.com/Quicr/new-qmedia/pull/72